relay_fragment_sender_test.go

```diff
@@ -0,0 +1,272 @@
+package tchannel
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/uber/tchannel-go/testutils/thriftarg2test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/uber/tchannel-go/typed"
```

Comment:
nit: import ordering. OK either either single group for everything, or break into 2 groups (one for tchannel, one for others). right now it's a mix